rns_shell/platform/graphics/x11/WindowX11.cpp

```diff
@@ -278,6 +278,15 @@ bool WindowX11::handleEvent(const XEvent& event) {
      *    a. shift ON  --> UpperCase
      *    b. shift OFF --> LowerCase
      */
+    if (event.type == KeyRelease && XEventsQueued(display_, QueuedAfterReading)) {
+        XEvent nev;
+        XPeekEvent(display_, &nev);
+        if (nev.type == KeyPress && nev.xkey.time == event.xkey.time &&
```

Comment:
I think following debug message is wrong or not clear. It enters this block in keypress but message says keyrelease.
Also, Can we add a comment here
// Unlike wpe backend, X11 sends both key press and release when key-repeat is on.  this code is for ignoring key press when keyrepeat is on

Reply:
addressed